interfaces/builtin/broadcom_asic_control.go

```diff
@@ -49,9 +40,11 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+# This file contains udev rules for accessing broadcom asic kernel module.
+#
+# Do not edit this file, it will be overwritten on updates
+
+SUBSYSTEM=="net", KERNEL=="bdev", TAG+="%[1]s"
```

Comment:
So long as we are using base_dev_name=bdev in the module loading, I guess what you had before would work. Where do we guarantee that is happening?

If using 'bdev' instead, can you add a comment above your rule?